article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -98,10 +99,25 @@ object ArticlePicker {
 
     if (tier == RemoteRender) {
       DotcomponentsLogger.logger.logRequest(s"path executing in dotcomponents", features, page)
+    } else if (tier == PressedArticle) {
+      DotcomponentsLogger.logger.logRequest(s"path executing from pressed content", features, page)
     } else {
       DotcomponentsLogger.logger.logRequest(s"path executing in web", features, page)
     }
 
     tier
   }
+
+  def decideTier(isPressed: Boolean, dcrCanRender: Boolean)(implicit
+      request: RequestHeader,
+  ): RenderType = {
+    if (request.forceDCROff) LocalRenderArticle
+    else if (isPressed) PressedArticle
+    else if (request.forceDCR || dcrCanRender) RemoteRender
```

Comment:
We also need to consider the semantics of the request saying `dcr=true` which is saying 'force this request to go to DCR' - it might be useful and more accurate if we respect this for pressed content?